.github/workflows/docs_workflow.yml

```
@@ -66,12 +66,14 @@ jobs:
           cd gh-pages
           # Create `.nojekyll` (if it doesn't already exist) for proper GH Pages configuration.
           touch .nojekyll
+          # Create an omega directory to store the documentation
+          mkdir omega
```

This will not cause an error if `omega` already exists.
```suggestion
          mkdir -p omega
```